tests/data_context/test_data_context.py

```diff
@@ -733,13 +734,25 @@ def basic_data_context_config():
         }
     })
 
+
 def test_ExplorerDataContext(basic_data_context_config):
     print(type(basic_data_context_config))
     ExplorerDataContext(
         basic_data_context_config,
         "testing/"
     )
 
+
+@freeze_time("2012-01-14")
```

Comment:
This feels like the kind of one-liner that could have taken a long time to figure out. Nifty pattern!